Chaincase.Common/Global.cs

```diff
@@ -343,7 +343,17 @@ public async Task<bool> WaitForInitializationCompletedAsync(CancellationToken ca
         private protected CancellationTokenSource ResumeCts { get; set; } = new CancellationTokenSource();
         private protected bool IsResuming { get; set; } = false;
 
-        public async Task OnResuming()
+        public void HandleRemoteNotification()
+        {
+            if (_walletManager?.SleepingCoins is { } && _torManager?.State != TorState.Started && _torManager.State != TorState.Connected)
+            {
+                _ = OnResuming(1);
+                // set timer = background time limit OR CoinJoin Complete
+                // sleep
+            }
+        }
```

Comment:
- [ ] check content of the notification for isCoinjoin

Reply:
do this at Chaincase.iOS/AppDelegate.cs, not here. this is iOS-specific behavior